### PR TITLE
[Enhancement] Add be_compactions to information_schema

### PR DESCRIPTION
**Pull request by meegoo**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

```
mysql> select * from information_schema.be_compactions;
+-------+----------------+-----------------------------+-----------------------------------+-----------------------+---------------------+-------------------------------+----------------------------------+
| BE_ID | CANDIDATES_NUM | BASE_COMPACTION_CONCURRENCY | CUMULATIVE_COMPACTION_CONCURRENCY | LAST_COMPACTION_SCORE | CANDIDATE_MAX_SCORE | MANUAL_COMPACTION_CONCURRENCY | MANUAL_COMPACTION_CANDIDATES_NUM |
+-------+----------------+-----------------------------+-----------------------------------+-----------------------+---------------------+-------------------------------+----------------------------------+
| 10005 |              0 |                           0 |                                 0 |                    24 |                   0 |                             0 |                                0 |
| 10004 |              0 |                           0 |                                 0 |                    24 |                   0 |                             0 |                                0 |
| 10006 |              0 |                           0 |                                 0 |                    24 |                   0 |                             0 |                                0 |
+-------+----------------+-----------------------------+-----------------------------------+-----------------------+---------------------+-------------------------------+----------------------------------+
3 rows in set (0.01 sec)
```
## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Review by gengjun-git — DISMISSED** [review]

File: be/src/exec/schema_scanner/schema_be_compactions_scanner.cpp [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+#include "util/starrocks_metrics.h"
+
+namespace starrocks {
+
+SchemaScanner::ColumnDesc SchemaBeCompactionsScanner::_s_columns[] = {
+        {"BE_ID", TYPE_BIGINT, sizeof(int64_t), false},
+        {"CANDIDATES_NUM", TYPE_BIGINT, sizeof(int64_t), false},
+        {"BASE_COMPACTION_CONCURRENCY", TYPE_BIGINT, sizeof(int64_t), false},
+        {"CUMULATIVE_COMPACTION_CONCURRENCY", TYPE_BIGINT, sizeof(int64_t), false},
+        {"LAST_COMPACTION_SCORE", TYPE_DOUBLE, sizeof(double), false},
```
[the remark this answers is not in the document]
Comment (meegoo) [reply be/src/exec/schema_scanner/schema_be_compactions_scanner.cpp:34]: The score of the latest compaction task

**Review by gengjun-git — DISMISSED** [review]

File: be/src/storage/compaction_manager.h [hunk 2]
```diff
@@ -133,6 +141,10 @@ class CompactionManager {
     int32_t _max_dispatch_count = 0;
 
     int32_t _max_task_num = 0;
+    int64_t _base_compaction_concurrency = 0;
+    int64_t _cumulative_compaction_concurrency = 0;
+    double _last_score = 0;
```
Comment (chaoyli) [inline be/src/storage/compaction_manager.h:146]: I can't get the idea of last_score. What's the benefit of the previous last score?

Comment (meegoo) [reply be/src/storage/compaction_manager.h:146]: Record the change of the latest compaction score, the candidate may be empty and cannot be replaced by the maximum score of the candidate.

**Review by gengjun-git — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**